### DISCUSSION TITLE
2 questions- Bit confused

### DISCUSSION THREAD

**Discussion by Darksidergr** [mriscoc/Ender3V2S1#1251] [Q&A]
I own a ender 3v2 with cr-touch ,4.2.2 board and 2.1.3 MRiscoc 20230312 

1. If I update the firmware to newer version what should I note down ? Is there any trick to keep settings after each update?

2. I don't understand the differnce between ubl and abl . I use 9x9 mesh and use the ( M420 S1 Z2 ; Use mesh level upto 2 mm) command on the slicer. Should I change to (G28 ; Home all axes
G29 L0 ; load a valid mesh from slot 0
G29 A  ; active the UBL system) 
and why?

sorry for my ignorance :)

**Comment by mriscoc** [comment]
Please check the firmware Wiki, some questions are already answered there.
